frontend/src/components/common/Game/Dialog/EditRom.vue

```diff
@@ -225,6 +225,14 @@ function closeDialog() {
     <template #append>
       <v-row class="justify-center mt-4 mb-2" no-gutters>
         <v-btn-group divided density="compact">
+          <v-btn
+            v-if="rom.igdb_id || rom.moby_id || rom.sgdb_id"
+            class="text-romm-red bg-terciary"
+            variant="flat"
+            @click="() => updateRom({ unmatch: true })"
```

Comment:
i think it would be cleaner to create a new `unmatchRom` function which just does:

```
await romApi
    .updateRom({ rom: rom.value, unmatch: true })
    .then...
```

Reply:
Done and added a screenshot